plugins/webp-uploads/picture-element.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * Add Picture Element support
+ * Description: Use <picture> element when image has more than one mime type.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Potentially wrap an image tag in a picture element.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $image         The image tag.
+ * @param string $context       The context for the image tag.
+ * @param int    $attachment_id The attachment id.
+ *
+ * @return string The new image tag.
+ */
+function webp_uploads_wrap_image_in_picture( string $image, string $context, int $attachment_id ): string {
+	if ( 'the_content' !== $context || ! webp_uploads_picture_element_enabled() ) {
```

Comment:
If the above suggestion is accepted:

```suggestion
	if ( 'the_content' !== $context ) {
```